apps/snitch_core/lib/core/data/schema/option_value.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Snitch.Data.Schema.OptionValue do
+  @moduledoc false
+  use Snitch.Data.Schema
+
+  alias Snitch.Data.Schema.OptionType
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_template_option_values" do
```

Comment:
1. Why is table name and schema name different?
2. Please follow our style, `timestamps()` at the bottom.

Reply:
Please rename the file as well.